test/access/manager/AccessManager.predicate.js

```diff
@@ -1,27 +1,23 @@
 const { setStorageAt } = require('@nomicfoundation/hardhat-network-helpers');
-const {
-  time: { setNextBlockTimestamp },
-} = require('@nomicfoundation/hardhat-network-helpers');
-const { time } = require('@openzeppelin/test-helpers');
-const { EXECUTION_ID_STORAGE_SLOT, EXPIRATION, scheduleOperation } = require('../../helpers/access-manager');
+const { EXECUTION_ID_STORAGE_SLOT, EXPIRATION, prepareOperation } = require('../../helpers/access-manager');
 const { impersonate } = require('../../helpers/account');
-const { expectRevertCustomError } = require('../../helpers/customError');
+const { bigint: time } = require('../../helpers/time');
+const { keccak256, AbiCoder } = require('ethers');
+const { ethers } = require('hardhat');
```

Comment:
It feels stang to import it twice from different sources. I'd either do
```suggestion
const { ethers, ethers: { keccak256, AbiCoder }} = require('hardhat');
```
or only import `ethers` and then use `ethers.keccak256` / `ethers.AbiCoder` everywhere

Reply:
It was only used once, so I applied your suggestion.